backend/api/auth.py

```diff
@@ -0,0 +1,39 @@
+from fastapi import APIRouter, Depends, HTTPException, status
+from ..models.auth import Token, TokenData, LoginData
+from ..services.auth import AuthService
+from ..services.team import TeamService
+
+
+__authors__ = ["Mustafa Aljumayli"]
+
+api = APIRouter(prefix="/api/auth")
+
+openapi_tags = {
+    "name": "Auth",
+    "description": "These routes are only used for demo/setup confirmation purposes.",
+}
+
+
+@api.post("/login", response_model=Token, tags=["Auth"])
+async def authenticate(
+    login_data: LoginData,
+    auth_service: AuthService = Depends(),
+    team_service: TeamService = Depends(),
+):
+    """Authenticates a team member and returns a JWT Token."""
+    team = team_service.get_team(login_data.name)
+    if not team:
+        raise HTTPException(
```

Comment:
A more idiomatic way to handle this is to create a custom exception in `services/exceptions.py` and define a custom exception handler in `main.py` (see the comments) so you can simply raise an exception in the service and have FastAPI take care of raising the HTTPError.

Reply:
I added that in🫡